app/assets/exercises/extract_method_5.rb

```diff
@@ -0,0 +1,40 @@
+def print_owing
```

Comment:
Why are these files in the assets folder and not under views, in templates or shared folder or something?